src/frontend/komponenter/Felleskomponenter/Skjemasteg/Skjemasteg.tsx

```diff
@@ -85,7 +78,16 @@ const Skjemasteg: React.FunctionComponent<IProps> = ({
             {children}
 
             <Navigering>
-                {nesteOnClick && skalViseNesteKnapp ? (
+                {forrigeOnClick && skalViseForrigeKnapp && (
```

Comment:
Hvordan blir tabbing nå? Grunnen til at vi brukte reverse er for at man skal tabbe rett til neste/bekreft knapp istedet for forrige.

Reply:
aha! Lurt. Da reverserer jeg :)